analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DeclareEventHandlersCorrectly.cs

```diff
@@ -53,11 +60,30 @@ public class ProviderChangedEventArgs : EventArgs {}
 
     public interface ILocalizationProvider
     {
-        event ProviderChangedEventHandler ProviderChanged; // Noncompliant
+        event ProviderChangedEventHandler ProviderChanged1; // Noncompliant
+        event ProviderChangedEventHandler ProviderChanged2; // Noncompliant
     }
 
-    public class Repro3453 : ILocalizationProvider
+    public abstract class BaseClass
     {
-        public event ProviderChangedEventHandler ProviderChanged; // Noncompliant (See https://github.com/SonarSource/sonar-dotnet/issues/3453)
+        public abstract event ProviderChangedEventHandler ProviderChanged3; // Noncompliant
+        public abstract event ProviderChangedEventHandler ProviderChanged4; // Noncompliant
+    }
+
+    public class Repro3453 : BaseClass, ILocalizationProvider
```

Comment:
could you please add a similar test, but with `IDisposable` instead of `ILocalizationProvider` ? 

This means that we'll ignore the issue whenever the containing class implements an interface? Isn't this too lax?

Shouldn't we check if the interface contains some event signatures?